hedera-node/src/main/java/com/hedera/services/state/exports/SignedStateBalancesExporter.java

```diff
@@ -96,6 +102,7 @@ public SignedStateBalancesExporter(
 	public boolean isTimeToExport(Instant now) {
 		if (periodEnd == NEVER) {
 			periodEnd = now.plusSeconds(dynamicProperties.balancesExportPeriodSecs());
+			return false;
```

Comment:
This return is redundant as line 112 returns the same.

Reply:
I was thinking to let it return `true` here to force it to do an export asap it restarts. I will change it that way.

Reply:
Renamed.